### DISCUSSION TITLE
如何优雅地在强化时只跳过白色航母

### DISCUSSION THREAD

**Discussion by guoh064** [LmeSzinc/AzurLaneAutoScript#2916] [Q&A]
我退役模式选择的是强化，但是也开了刷紧急委托。我希望在打主线图/活动图的时候不强化白色航母，但强化剩下的所有船，有什么比较优雅的处理方法吗，谢谢！

**Comment by bookbug666** [comment]
我想到的辦法是先退役，但是由alas接管退役選項排除白皮(避免把強化素材分解了)，並且在一次退役後還原原本的選項配置
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/29662367/396bf510-5ca0-471b-8225-f12cae77bc00)
而後排除稀有度N的船，如圖
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/29662367/b0535454-b490-453b-9290-7f321cb07006)

只有先退役才能排除先強化藍紫皮的問題，短暫關閉拆白皮才不會讓素材在一次退役中消失
不知道這算不算優雅，還不夠優雅的話可能要做新的傳統退役了

**Reply by bookbug666** [reply]
But讓alas去接管用戶配置的話會有些風險，還需要做一些防呆
而且變更用戶設置也可能會因為UI變動而無意使用戶選項變成拆到他不想拆的，這是不太優秀的一點
